src/main/java/uk/gov/hmcts/reform/sscscorbackend/domain/AddressDetails.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.sscscorbackend.domain;
+
+import com.fasterxml.jackson.annotation.*;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.Objects;
+
+@JsonInclude(value = JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AddressDetails {
+    private String line1;
+    private String line2;
+    private String town;
+    private String county;
+    private String postcode;
+
+    public AddressDetails() {
+    }
```

Comment:
Tiny thing but I do you need this no arg constructor. Is this object ever created from JSON, I think its only used the other way where we create the json from the object so we don't need it. Then all the fields could be final too.